### ISSUE TITLE
How to change column value depending on selection of other field using relationship field

### ISSUE THREAD

**Issue by rodrane** [FrozenNode/Laravel-Administrator#1054]
Hi,
I let user to choose participation values from panel.
Depending on selection I have to change columns is_participant to 1

this is my field on [AdminPanel][1]

    'participant' => [
                'title' => 'administrator.fields_watcher',
                'type' => 'relationship',
                'name_field' => 'name',
                'options_sort_field' => 'name',    
    ],

This is my model file

     public function challenges()
        {
            return $this->belongsToMany(Challenge::class, 'role_challenge', 'role_id', 'challenge_id');
        }

     public function participant()
        {
            return $this->challenges()->where('is_participant','=',TRUE)
            ->where('is_watcher','=',TRUE);
        }

[![enter image description here][2]][2]

And this is my database table 

[![enter image description here][3]][3]

So basically when user choose a challenge (test1, test2, test3)  to **participant field** I want to change **is_participant** value to **1** in database if it is not exist will add new line with is_participant value is 1. 

I can add new line when I choose (test1, test2,test3) but I cannot change column values. 

thank you 

  [1]: https://github.com/rodrane/Laravel-Administrator/ "Admin Panel"
  [2]: https://i.stack.imgur.com/5bCmI.png
  [3]: https://i.stack.imgur.com/16dEn.png

**Comment by rodrane** [comment]
@janhartigan
